File: js/first-mms.js
const twilio = require('twilio');

// authentication Api
const client = twilio(
	// I liek to make sure things are the right type.
	String(process.env.ACCOUNT_SID),
	String(process.env.ACCOUNT_TOKEN)
);

client.messages
	.create({
		from: String(process.env.NUMBER_FROM),
		to: String(process.env.NUMBER_TO),
		// Input the "to" number here
		// What is the mediaUrl that you want to send in your message?
		// Why don't you add a body parameter too?
	})
	.then(message => {
		console.log(
			'Copy this MMS SID. How does it look different from an SMS SID?'
		);
		console.log(`${message.sid}`);
	})
	.catch(error => {
		console.error('Looks like the Twilio API returned an error:');
		console.error(error);
	});

File: py/first-mms.py
from twilio.rest import Client
from decouple import config

ACCOUNT_SID = config('ACCOUNT_SID')
ACCOUNT_TOKEN = config('ACCOUNT_TOKEN')
NUMBER_FROM = config('NUMBER_FROM')
NUMBER_TO = config('NUMBER_TO')
# your account sid and auth token from twilio.com/console
account_sid = ACCOUNT_SID
auth_token = ACCOUNT_TOKEN
client = Client(account_sid, auth_token)

message = client.messages.create(
    media_url="https://api.assistivecards.com/cards/shapes/heart@2x.png",
    body="This is a heart.",
    from_=NUMBER_FROM,
    to=NUMBER_TO
)

print(message.sid)
